LocalSettings.php

```diff
@@ -3747,6 +3757,11 @@
 			'editrestrictedtemplateprotected',
 			'editimportprotected',
 		],
+    		'allpediawiki' => [
```

Comment:
This is the wrong array syntax for here. Would  drop the first array level, see others for example.

Reply:
@Universal-Omega Huh? This is the way I've done it before, three times, following others' lead, and Reception123's merged them each time? I believe this is correct.

Reply:
There is no other like this way.

Reply:
> There is no other like this way. Not for this config.

If this is the bottom variable, yes, they're all like that.

Reply:
```php
'famepediawiki' => [	'editextendedconfirmedprotected',
'edittemplateprotected',
],
``` snippet of another. Which doesn't match yours.

Reply:
@Universal-Omega Yeah I see it. That's a copy/paste mistake on my part. My bad. ;)

Reply:
Should be done now. :)